docs/intro/tutorial08.txt

```diff
@@ -0,0 +1,135 @@
+<https://jazzband.co>`_ organization. The toolbar helps you understand how your
+application functions and to identify problems. It does so by providing panels
+that provide information about a specific system of Django applications.
+
+- Cache panel: Presents you with the cache operations made and the context
+  of the executions.
+- Profiler panel: Presents you with a `cProfiler
+  <https://docs.python.org/3/library/profile.html>`_ summary of the codes
+  execution.
+- Request panel: Presents you with the headers and body of a request. This
+  allows you to determine what values are being sent to the Django application.
+- SQL panel: Presents you with all
+  the SQL queries made during the lifespan of a request.
+- Templates panel: Presents you with the templates used when rendering the
+  response and the context used to render them.
+
+To install a third-party application like the toolbar, you need to install
+the package by running the command within an activated virtual environment.
+This is like our earlier step of :ref:`installing an official release of
+Django <installing-official-release>`.
+
+   .. console::
+
+        $ python -m pip install django-debug-toolbar
+
+
+.. admonition:: Managing application dependencies
+
+    Managing dependencies is an important part developing an application, but
+    it is outside of the scope of this tutorial. Please see the official Python
+    documentation regarding `Managing Application Dependencies
+    <https://packaging.python.org/en/latest/tutorials/managing-dependencies/>`_
+    for more information.
+
+Third-party packages that integrate with Django may need some setup.
+Sometimes you will need to add the package's Django app to your
+:setting:`INSTALLED_APPS` setting. `The Django Debug Toolbar has many
+installation steps
+<https://django-debug-toolbar.readthedocs.io/en/latest/installation.html>`_.
+We're linking to the toolbar's documentation here because as a third-party
+package, it's maintained outside of the core development of Django. This
+allows the package maintainers to make releases outside of Django's release
+schedule
+
+Once installed, you should be able to see the DjDT "handle" on the right side
+of the browser window when you refresh the polls application.
+
+Getting help from others
+========================
+
+At some point you will run into a problem, for example the
+toolbar may not render. When this happens and you're unable to
+resolve the issue yourself, there are options available to you.
+
+#. If the problem is with a specific package, check if there's a
+   troubleshooting of FAQ in the package's documentation. For example the
+   Django Debug Toolbar has a `Tips section
+   <https://django-debug-toolbar.readthedocs.io/en/latest/tips.html>`_ that
+   outlines troubleshooting options.
+#. Consult the `Django Forum <https://forum.djangoproject.com/>`_.
+#. Join the `Django Discord server <https://discord.me/unofficial-django>`_.
```

Comment:
I think offical docs should like to the official discord.

Reply:
@knyghty do you know if we can rename the discord server? Otherwise @smithdc1 we can use the invite link.

Reply:
I have no idea about renaming or why it'd be needed here...? I think using an invite link is fine for now. Ideal would be to link to the landing page on discord.com - but we can't have one of those until someone (not me) makes a 1920x1080 cover image.

Reply:
Sorry, I included you since you are the admin on discord adding a bot.

For your suggestion of linking to the landing page, do you mean discord.com? Then the user would search for Django?

Right now the server is still called "unofficial django". The landing page for our discord server has that in the URL. Or if the user searches for it, it's named that. Since the server is now operating under the django CoC, I thought it was official. If it is, then we should probably rename it.

In the context of this PR, it's being used to indicate where else people can get help. If we can't rename it, I'll use the invite link.

Reply:
Ah, ok, yeah, that isn't the same server as ours, that is indeed an unofficial one. Ours is just called Django. I don't think discord.me is anything official, so not sure where it's getting its data.

Reply:
I switched this to the invite link we use on djangoproject.com/